src/main/scala/tile/CustomCSRs.scala

```diff
@@ -38,6 +38,7 @@ class CustomCSRs(implicit p: Parameters) extends CoreBundle {
   def disableICacheClockGate = getOrElse(chickenCSR, _.value(1), true.B)
   def disableCoreClockGate = getOrElse(chickenCSR, _.value(2), true.B)
   def disableSpeculativeICacheRefill = getOrElse(chickenCSR, _.value(3), true.B)
+  def suppressCorruptOnGrantData = getOrElse(chickenCSR, _.value(9), true.B)
```

Comment:
i just realized that if we DON'T have the chickenCSR, that means that all these bits are hard-coded to true? I guess I would have expected the inverse.

Reply:
but otherwise given that this seems fine to me

Reply:
Uh, I guess you're right.  It hasn't been an issue since the chickenCSR is always present in Rocket cores.  Might as well fix this for clarity.